pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/secretmanagersecretversion/update.yaml

```diff
@@ -17,7 +17,6 @@ kind: SecretManagerSecretVersion
 metadata:
   annotations:
     cnrm.cloud.google.com/project-id: ${projectId}
-    alpha.cnrm.cloud.google.com/reconciler: direct
```

Comment:
I guess technically we can keep these and then have a separate "cleanup redundant annotations" bot that goes in and removes them.  I feel less strongly about this one than the idea of keeping the `-legacy` tests though, a few one line changes are not a big deal